### PR TITLE
Add CN doc of Algolia search

### PR DESCRIPTION
**Pull request by sli1989**
## PR Checklist
**Please check if your PR fulfills the following requirements:**

- [x] The commit message follows [our guidelines](https://github.com/theme-next/hexo-theme-next/blob/master/.github/CONTRIBUTING.md).
- [x] Tests for the changes have been added (for bug fixes / features).
   - [x] Muse | Mist have been tested.
   - [ ] Pisces | Gemini have been tested.
- [x] Docs have been added / updated (for bug fixes / features).

## PR Type
**What kind of change does this PR introduce?**  

- [ ] Bugfix.
- [ ] Feature.
- [ ] Code style update (formatting, local variables).
- [ ] Refactoring (no functional changes, no api changes).
- [ ] Build related changes.
- [ ] CI related changes.
- [x] Documentation content changes.
- [ ] Other... Please describe:

## What is the new behavior?

According to issue [#68](https://github.com/theme-next/hexo-theme-next/issues/68), Add CN doc of Algolia search to install it successfully in NEXT V6+.

### CODE REVIEW

**Comment by ivan-nginx** [comment]
Can't read this. Please, add EN docs to possible read and translate it on other languages.

**Comment by sli1989** [comment]
i am working on en doc.

File: docs/ALGOLIA-SEARCH.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+      applicationID: 'Application ID'
+      apiKey: 'Search-only API key'
+      indexName: 'indexName'
+      chunkSize: 5000
+    ```
+
+1. Run the following command to upload index data, keep a weather eye out the output of the command.
+
+    ```
+    $ export(windows 为 set) HEXO_ALGOLIA_INDEXING_KEY=Search-Only API key
```
Comment (ivan-nginx) [inline docs/ALGOLIA-SEARCH.md]: 为 — what it this?

File: docs/ALGOLIA-SEARCH.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+1. Algolia requires users to upload their search index data either manually or via provided APIs. Install and configure [Hexo Algolia](https://github.com/oncletom/hexo-algolia) in your Hexo directory. This plugin will index your site and upload selected data to Algolia.
+
+    ```
+    cd hexo
+    npm install --save hexo-algolia
+    ```
+
+1. Go to API Keys and find your credentials. You will need the `Application ID`, the `Search-only API key` and the `Admin API key` in the following sections. The `Admin API key` need to keep confidential.
+
+    ![          ](http://theme-next.iissnan.com/uploads/algolia/algolia-step-3.png)
```
Comment (ivan-nginx) [inline docs/ALGOLIA-SEARCH.md]: Why space?

Comment (sli1989) [reply docs/ALGOLIA-SEARCH.md]: oh, it's a insertion error. 
i will fix it.

File: docs/ALGOLIA-SEARCH.md (hunk 1 above)
Comment (sli1989) [reply docs/ALGOLIA-SEARCH.md]: sorry for the mistake.
it's supported to be

```
export HEXO_ALGOLIA_INDEXING_KEY=Search-Only API key # Git Bash
set HEXO_ALGOLIA_INDEXING_KEY=Search-Only API key # Windows command line (DOS) 
```

**Pull request merged**